src/components/EkoVideo/EkoVideo.jsx

```diff
@@ -81,6 +84,9 @@ export function EkoVideo({
             if (onPlayerInit){
                 onPlayerInit(playerRef.current);
             }
+            if (context && context.setPlayerState) {
```

Comment:
out of curiosity in which cases will context not be truthy?

Reply:
Ah exactly in the cases where the context *value* was not provided without a wrapper. Got it